checker/raw_result.go

```diff
@@ -125,10 +125,35 @@ type MaintainedData struct {
 	ArchivedStatus       ArchivedStatus
 }
 
+type LicenseAttributionType string
+
+const (
+	// forms of security policy hints being evaluated.
+	LicenseAttributionTypeOther     LicenseAttributionType = "otherAttribution"
```

Comment:
Do you think we need this level of granularity to be exposed? It feels like an implementation detail to me, and consumers should not have to know about it. I'm worried that exposing this would cause breaking changes as soon as the implementation changes. I would suggest we remove it. Wdut?

Reply:
I still do and I am returning this information to the evaluation for comprehension in the scoring.